cent/exceptions.py

```diff
@@ -0,0 +1,38 @@
+from cent.methods.base import CentMethod, CentType
+
+
+class CentError(Exception):
```

Comment:
Looks like now transport level exceptions and timeout exceptions from Session implementation will be raised untouched right? Most probably it's OK - but need to properly document I suppose. WDYT?